cowprotocol/accounting/rewards/mainnet/service_fee_query_DEPRECATED_4017925.sql

```diff
@@ -1,3 +1,7 @@
+------------ DEPRECATED!
```

Comment:
Adding this comment is useful. Keeping the file under version control probably not.

It the goal of this to allow for a non-breaking transition?

Reply:
Yeah good point. I guess in order to add this comment, we either need to first merge this PR and then have a separate one delete the relevant file, or manually add this comment directly on Dune.

Dune doesn't allow the deletion of queries, right?

Reply:
One can _archive_ queries. I think such queries cannot be used from within other queries.